cfgov/v1/forms.py

```diff
@@ -241,11 +244,8 @@ def set_interpreted_date_values(self, cleaned_data):
                     earliest_results, self.preferred_datetime_format)
 
             if to_date:
-                transformed_to_date = end_of_time_period(
-                    self.data['to_date'], cleaned_data['to_date'])
-                cleaned_data['to_date'] = transformed_to_date
                 self.data['to_date'] = date.strftime(
-                    transformed_to_date, self.preferred_datetime_format)
+                    cleaned_data['to_date'], self.preferred_datetime_format)
```

Comment:
It looks like [`end_of_time_period` was added to handle ](https://github.com/cfpb/consumerfinance.gov/commit/624c5ae4ad2dddf0b30bb671a3467cc08d017181#diff-40757824786a8c38156d12b1db8227629193ded9e4279342230a81d3e4844250) ambiguous dates. Are those still handled?

Reply:
@willbarton I can add more tests to check for ambiguous dates to verify that they are handled.
Do you have any specific examples in mind?

Reply:
Good catch. I added `end_of_time_period` to handle ambiguous dates.